src/java/com/zimbra/graphql/resolvers/impl/AccountResolver.java

```diff
@@ -96,4 +98,16 @@ public void accountEndSession(
         @GraphQLRootContext RequestContext context) throws ServiceException {
         return accountRepository.prefsModify(context, prefs);
     }
+
+    @GraphQLMutation(description="Change password.")
+    public ChangePasswordResponse changePassword(
```

Comment:
Are we better off returning an `AuthToken`, or creating a `GQLAuthToken` class with the annotation `name="AuthToken"` instead of returning the entire response class?